defaults/cslc_s1.yaml

```
@@ -0,0 +1,45 @@
+runconfig:
+   name: cslc_s1_workflow_default
+
+   groups:
+       pge_name_group:
+           pge_name: CSLC_S1_PGE
+
+       input_file_group:
+           # Required. List of SAFE files (min=1)
+           safe_file_path:
+           # Required. List of orbit (EOF) files (min=1)
+           orbit_file_path:
+           # List of (unique) burst IDs to process
+           burst_id:
+           # Options for reference burst
+           reference_burst:
+               is_reference: False
+               file_path:
```

```suggestion
               is_reference: True 
               file_path:
```
What do you think about making the default `is_reference` to `True` and assume any user starting from nothing? Also allows us to not have to worry about what's the correct default `file_path` for non-reference cases.

---

Sounds good to me. I will apply this change